src/pvecontrol/actions/cluster.py

```diff
@@ -33,21 +35,54 @@ def _get_disk_output():
         d_percent = metrics["disk"]["percent"]
         return f"{d_usage}/{d_total}({d_percent:.1f}%)"
 
-    output = f"""
-  Status: {status}
-  VMs: {vms - templates}
-  Templates: {templates}
-  Metrics:
-    CPU: {_get_cpu_output()}
-    Memory: {_get_memory_output()}
-    Disk: {_get_disk_output()}
-  Nodes:
-    Offline: {len([node for node in proxmox.nodes if node.status == NodeStatus.OFFLINE])}
-    Online: {len([node for node in proxmox.nodes if node.status == NodeStatus.ONLINE])}
-    Unknown: {len([node for node in proxmox.nodes if node.status == NodeStatus.UNKNOWN])}
-  """
-
-    print(output)
+    if _args.output == OutputFormats.TEXT:
+        output = f"""\
+            Status: {status}
+            VMs: {vms - templates}
+            Templates: {templates}
+            Metrics:
+                CPU: {_get_cpu_output()}
+                Memory: {_get_memory_output()}
+                Disk: {_get_disk_output()}
+            Nodes:
+                Offline: {len([node for node in proxmox.nodes if node.status == NodeStatus.OFFLINE])}
+                Online: {len([node for node in proxmox.nodes if node.status == NodeStatus.ONLINE])}
+                Unknown: {len([node for node in proxmox.nodes if node.status == NodeStatus.UNKNOWN])}
+            """
+        print(textwrap.dedent(output))
+    else:
+        render_table = [
+            dict(
+                status=status,
+                vm=vms - templates,
+                templates=templates,
+                metrics=dict(
+                    cpu=dict(
+                        usage=metrics["cpu"]["usage"],
+                        total=metrics["cpu"]["total"],
+                        percent=metrics["cpu"]["percent"],
+                        allocated=metrics["cpu"]["allocated"],
+                    ),
+                    memory=dict(
+                        usage=metrics["memory"]["usage"],
+                        total=metrics["memory"]["total"],
+                        percent=metrics["memory"]["percent"],
+                        allocated=metrics["memory"]["allocated"],
+                    ),
+                    disk=dict(
+                        usage=metrics["disk"]["usage"],
+                        total=metrics["disk"]["total"],
+                        percent=metrics["disk"]["percent"],
+                    ),
+                ),
```

Comment:
This seems a bit repetitive. Couldn't we do `metrics=metrics`?

Reply:
oh :face_exhaling: